### ISSUE TITLE
Allow for email addresses containing '$'

### ISSUE THREAD

**Issue by beirtipol** [johnsusek/praeco#283]
My company uses the '$' symbol to prefix group emails. Unfortunately that doesn't seem to be supported by Praeco/ElastAlert :/

**Comment by nsano-rururu** [comment]
You pointed out that it is not a validation that allows characters that can be used in the local part. I'd like to check with manishsaraan/email-validator, but do you know any other recommended libraries?
https://github.com/manishsaraan/email-validator

Line 599
https://github.com/johnsusek/praeco/blob/master/src/components/config/alert/ConfigAlert.vue#L599

```
  if (!value || value.match(/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i)) {
```

Line 631
https://github.com/johnsusek/praeco/blob/master/src/components/config/alert/ConfigAlert.vue#L631


```
      && !email.trim().match(/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i)
```

**Comment by beirtipol** [comment]
Sorry, I'm not aware of any libraries which would be any better. It's a pretty common issue we face as I don't think it's standard to use '$' in email addresses.

**Comment by nsano-rururu** [comment]
It was stated in RFC5322 that alphanumeric characters and! # $% &'* +-/ =? ^ _ `{|} ~" Can be used immediately before @. Check the operation of the library and the target character is If it's allowed, I'll add the library, if not, I'll modify my current proprietary checking process.

**Comment by nsano-rururu** [comment]
We apologize for any inconvenience.

**Comment by beirtipol** [comment]
This seems to work:

var validator = require("email-validator");

console.log( validator.validate("$myemail@email.com")); //prints 'true'

**Comment by nsano-rururu** [comment]
Thank you for trying.
Adopt email-validator.

**Comment by nsano-rururu** [comment]
Released a version "praecoapp / praeco: 1.5.1" with a modified email check
https://hub.docker.com/r/praecoapp/praeco
